Content.Server/StationEvents/RampingStationEventSchedulerSystem.cs

```diff
@@ -61,17 +66,33 @@ public override void Update(float frameTime)
                 return;
             }
 
-            PickNextEventTime(uid, scheduler);
+            PickNextEventTime(scheduler);
             _event.RunRandomEvent();
         }
     }
 
-    private void PickNextEventTime(EntityUid uid, RampingStationEventSchedulerComponent component)
+    private void PickNextEventTime(RampingStationEventSchedulerComponent component)
     {
+        // In case of server hosts being silly and setting maximum time to be lower than minimum time, sanity check the scheduler inputs and sort them by Min/Max
+        var minimumTime = MathF.Min(_cfg.GetCVar(CCVars.GameEventsRampingMinimumTime)
+                                        - _cfg.GetCVar(CCVars.GameEventsRampingMinimumTimeOffset)
+                                        - component.MinimumEventTimeOffset,
+                                    _cfg.GetCVar(CCVars.GameEventsRampingMaximumTime)
+                                        - _cfg.GetCVar(CCVars.GameEventsRampingMaximumTimeOffset)
+                                        - component.MaximumEventTimeOffset);
+
+        var maximumTime = MathF.Max(_cfg.GetCVar(CCVars.GameEventsRampingMinimumTime)
+                                        - _cfg.GetCVar(CCVars.GameEventsRampingMinimumTimeOffset)
+                                        - component.MinimumEventTimeOffset,
+                                    _cfg.GetCVar(CCVars.GameEventsRampingMaximumTime)
+                                        - _cfg.GetCVar(CCVars.GameEventsRampingMaximumTimeOffset)
+                                        - component.MaximumEventTimeOffset);
```

Comment:
```suggestion
        // In case of server hosts being silly and setting maximum time to be lower than minimum time, sanity check the scheduler inputs and sort them by Min/Max
        var minimumTime = MathF.Min(_cfg.GetCVar(CCVars.GameEventsRampingMinimumTime)
	        - _cfg.GetCVar(CCVars.GameEventsRampingMinimumTimeOffset)
	        - component.MinimumEventTimeOffset, _cfg.GetCVar(CCVars.GameEventsRampingMaximumTime)
	        - _cfg.GetCVar(CCVars.GameEventsRampingMaximumTimeOffset)
	        - component.MaximumEventTimeOffset);

		var maximumTime = MathF.Max(_cfg.GetCVar(CCVars.GameEventsRampingMinimumTime)
	        - _cfg.GetCVar(CCVars.GameEventsRampingMinimumTimeOffset)
	        - component.MinimumEventTimeOffset, _cfg.GetCVar(CCVars.GameEventsRampingMaximumTime)
	        - _cfg.GetCVar(CCVars.GameEventsRampingMaximumTimeOffset)
	        - component.MaximumEventTimeOffset);
```